nautobot/project-static/js/job_result.js

```diff
@@ -1,42 +1,58 @@
 var url = nautobot_api_path + "extras/job-results/";
 var timeout = 1000;
+var terminal_statuses = ['completed', 'failed', 'errored'];
 
-function updatePendingStatusLabel(status){
+function updatePendingStatusLabel(status) {
+    // Updates "Status" label in "Summary of Results" table in JobResult detail view.
     var labelClass;
-    if (status.value === 'failed' || status.value === 'errored'){
+    if (status.value === 'failed' || status.value === 'errored') {
         labelClass = 'danger';
-    } else if (status.value === 'running'){
+    }
+    else if (status.value === 'running') {
         labelClass = 'warning';
-    } else if (status.value === 'completed'){
+    }
+    else if (status.value === 'completed') {
         labelClass = 'success';
-    } else {
+    }
+    else {
         labelClass = 'default';
     }
     var elem = $('#pending-result-label > label');
     elem.attr('class', 'label label-' + labelClass);
     elem.text(status.label);
 }
 
-function refreshWindow(){
-    window.location.reload();
+function updateLogTable(result_id) {
+    // Calls `update_log_table` to refresh the jobs table from the `/log-table/` endpoint
+    update_log_table('', '/extras/job-results/' + result_id + '/log-table/');
```

Comment:
For my education, where does `update_log_table` come from? Is this some magic from django-ajax-tables?

Reply:
It comes from `django-ajax-tables`. I had that in there before and removed it.

Reply:
Specifically: https://github.com/KiwiTrinsic/django-ajax-tables/blob/master/django_ajax_tables/templates/django_ajax_tables/ajax_wrapper.html#L18